### PR TITLE
fix windows build

### PR DESCRIPTION
**Pull request by Dimon4eg**
It builds but have not tested it yet :)

### CODE REVIEW

**Comment by Dimon4eg** [comment]
@bsergean can we add windows build to CI?

File: ws/CMakeLists.txt [hunk 1]
```diff
@@ -51,4 +53,13 @@ add_executable(ws
   ws.cpp)
 
 target_link_libraries(ws ixwebsocket)
+
```
Comment (bsergean) [inline ws/CMakeLists.txt]: Does that one means that we need OpenSSL on Windows ?
Could we make that one use the TLS macro, or be Linux only ?

**Comment by bsergean** [comment]
Yes it would be good to have Windows setup on CI, but Windows platform wasn't available on travis when I started this.

https://blog.travis-ci.com/2018-10-11-windows-early-release
https://docs.travis-ci.com/user/reference/windows/

In the forums they explain how to setup msvc by running some batch files.

File: ws/CMakeLists.txt (hunk 1 above)
Comment (Dimon4eg) [reply ws/CMakeLists.txt]: we don't build we on windows currently as there are many errors.

**Pull request merged**